.pylintrc

```diff
@@ -61,7 +61,7 @@ confidence=
 # --disable=W"
 #disable=import-star-module-level,old-octal-literal,oct-method,print-statement,unpacking-in-except,parameter-unpacking,backtick,old-raise-syntax,old-ne-operator,long-suffix,dict-view-method,dict-iter-method,metaclass-assignment,next-method-called,raising-string,indexing-exception,raw_input-builtin,long-builtin,file-builtin,execfile-builtin,coerce-builtin,cmp-builtin,buffer-builtin,basestring-builtin,apply-builtin,filter-builtin-not-iterating,using-cmp-argument,useless-suppression,range-builtin-not-iterating,suppressed-message,no-absolute-import,old-division,cmp-method,reload-builtin,zip-builtin-not-iterating,intern-builtin,unichr-builtin,reduce-builtin,standarderror-builtin,unicode-builtin,xrange-builtin,coerce-method,delslice-method,getslice-method,setslice-method,input-builtin,round-builtin,hex-method,nonzero-method,map-builtin-not-iterating
 
-disable=superfluous-parens,locally-disabled,misplaced-comparison-constant,too-few-public-methods,no-self-use,invalid-sequence-index
+disable=superfluous-parens,locally-disabled,misplaced-comparison-constant,too-few-public-methods,no-self-use,invalid-sequence-index,cyclic-import
```

Comment:
Really hoping to avoid doing this. Everything I read about cyclic imports is that they often indicate it's time to refactor. For a long time I've been complaining about the confusing inheritance among `Model`, `ConstraintSet`, and `CostedConstraintSet`. I think fixing the circular imports issue is a chance to clean that up, and disabling this will make us ignore it. GPkit's use of inheritance is fairly well structured, let's not give up on making it nice at the top level.